src/design-system/playground/BackgroundDemo.tsx

```diff
@@ -9,23 +9,23 @@ export function BackgroundDemo() {
   const { backgroundColors } = useColorMode();
 
   return (
-    <>
-      {(Object.keys(backgroundColors) as (keyof typeof backgroundColors)[])
-        .sort()
```

Comment:
This sort was removed so that we can choose the order of colors, which also helps ensure that deprecated values are at the bottom.